pkg/zcmd/zpool/import/builder.go

```diff
@@ -165,11 +165,11 @@ func (p *PoolImport) Build() (*PoolImport, error) {
 	var c strings.Builder
 	p = p.Validate()
 	p.appendCommand(&c, bin.ZPOOL)
-	p.appendCommand(&c, fmt.Sprintf(" %s ", Operation))
+	p.appendCommand(&c, " "+Operation+" ")
 
 	if IsPropertySet()(p) {
 		for _, v := range p.Property {
-			p.appendCommand(&c, fmt.Sprintf(" -o %s ", v))
+			p.appendCommand(&c, " "+v+" ")
```

Comment:
To set zpool property `-o` (option) is required but we are missing in above change